kuma/notifications/utils.py

```diff
@@ -75,8 +77,29 @@ def pluralize(browser_list):
 }
 
 
+def publish_content_notification(url, text, dry_run=False):
+    watchers = Watch.objects.filter(url=url)
+
+    if not watchers:
+        return
+
+    if dry_run:
```

Comment:
Is dry_run something we are interested in maintaining?

Reply:
Fixed in 049a72725.